scripts/build-upload-docker-images.sh

```diff
@@ -2,16 +2,19 @@
 
 set -euxf -o pipefail
 
+# build multi-arch docker images by default
+platforms="${PLATFORMS:-linux/amd64,linux/s390x,linux/ppc64le,linux/arm64}"
```

Comment:
You're not really using the value of env var effectively, so instead of trying to pass full platform name and then doing ugly check in L13 you can pass an argument to the script to indicate pr-only mode and used that to decide which commands to execute and which platforms to build.

Reply:
Thx for reviewing! @yurishkuro 
Your suggestion sounds  great.
Not sure if understood, but I tried passing the `pr-only` arg to the script.
Would that be good?